src/util.ts

```diff
@@ -131,20 +131,12 @@ function removeManifest(application: string, manifestPath: string): Promise<any>
 
 // NON-WIN32 COMMANDS //
 function addManifestToSideloadingDirectory(application: string, manifestPath: string): Promise<any> {
-  return new Promise ((resolve, reject) => {
+  return new Promise (async (resolve, reject) => {
```

Comment:
I don't think it is a good idea to mix code that returns new Promise() with an async function. 

I don't even see any code that needs to declare the promise. Instead, I believe you can make it `async function addManifestToSideloadingDirectory` and removing the `return new Promise`. When the function completes, it will then resolve. 

This should also be Promise<void> as I only see `resolve()` in the code now and no actual value returned via the promise.